lib/go/edgecontext/backend.go

```diff
@@ -3,11 +3,18 @@ package edgecontext
 import (
 	"context"
 	"fmt"
+	"sync/atomic"
 
 	"github.com/reddit/baseplate.go/ecinterface"
 	v0 "github.com/reddit/edgecontext/lib/go/internal/v0"
 )
 
+func init() {
+	SetBackend(defaultBackend())
+}
+
+var backend atomic.Pointer[*Backend]
```

Comment:
you want:
```suggestion
var backend atomic.Pointer[Backend]
```
instead to avoid double pointer 😅

Reply:
Ya, this felt weird